packages/evm-connector/vite.config.ts

```diff
@@ -0,0 +1,31 @@
+// vite.config.js
+import { resolve } from 'node:path';
+import { defineConfig } from 'vite';
+import dts from 'vite-plugin-dts';
+
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+export default defineConfig({
```

Comment:
it doesn't look we're using any vite app around this PR, did I miss something?
could we delete this file and vite package?

Reply:
Without this file the `beforeAll` in [the test file](https://github.com/FuelLabs/fuel-connectors/blob/56b18b355919c3f9e0fa4ad2172032d334530992/packages/evm-connector/src/test/walletConnector.test.ts#L51) times out due to `__dirname` being undefined.
Additionally, as I understand it the following provides access to `window`:
```
test: {
    environment: 'jsdom',
  },
```
Is there another file where I should be doing this instead?